### PR TITLE
Gibs docs

### PR DESCRIPTION
**Pull request by egarreau**
## Description of change
Adding documentation to GIBS (one of the EVSS services)

## Acceptance Criteria (Definition of Done)
Documentation is added

#### Unique to this PR

- [x] yard docs are added to all relevant classes

#### Applies to all PRs

- [x] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: lib/evss/gi_bill_status/amendment.rb [hunk 1]
```diff
@@ -4,6 +4,22 @@
...
   module GiBillStatus
+    ##
+    # Model for an amendment made to a user's enrollment
+    #
+    # @!attribute on_campus_hours
+    #   @return [Float] The number of credit hours the user was on campus
+    # @!attribute online_hours
+    #   @return [Float] The number of credit hours the user took online
+    # @!attribute yellow_ribbon_amount
+    #   @return [Float] The institution's contributions to the Yellow Ribbon Program
```
Comment (kfrz) [inline lib/evss/gi_bill_status/amendment.rb]: Just to be certain, this is financial contributions in cents, right?

File: lib/evss/gi_bill_status/configuration.rb [hunk 2]
```diff
@@ -2,15 +2,28 @@
...
 
+      ##
+      # @return [String] Service name to use in breakers and metrics.
+      #
       def service_name
         'EVSS/GiBillStatus'
       end
 
+      ##
+      # @return [Boolean] Should the service use mock data in lower environments.
```
Comment (kfrz) [inline lib/evss/gi_bill_status/configuration.rb]: Will duplicate language in other documented service/**/configuration.rb files

File: lib/evss/gi_bill_status/amendment.rb (hunk 1 above)
Comment (egarreau) [reply lib/evss/gi_bill_status/amendment.rb]: Great question; I'm actually not sure. Looking into it now

File: lib/evss/gi_bill_status/configuration.rb (hunk 2 above)
Comment (egarreau) [reply lib/evss/gi_bill_status/configuration.rb]: How would you suggest avoiding this duplication?

Comment (kfrz) [reply lib/evss/gi_bill_status/configuration.rb]: Sorry I should have been more specific - I like this language and I think we should use it consistently for all of the service `configuration.rb` files. Probably didn't need to add the comment here :) :shipit: :fist_oncoming:

**Pull request merged**